### ISSUE TITLE
Access keys not working as expected with custom titleBarStyle

### ISSUE THREAD

**Issue by rik-smeets** [Microsoft/vscode#54349]
- VSCode Version: 1.26.0 Insider (last commit date 2018-07-13)
- OS Version: Windows 10 April 2018 Update

**Steps to Reproduce**:

1. Ensure `Underline keyboard shortcuts and access keys` is checked in Windows' *Make the keyboard easier to use* options, under *Ease of Access* in the control panel.
2. In user settings, set the titleBarStyle to "custom" (`"window.titleBarStyle": "custom"`).
*Note: with the titleBarStyle set to `native`, keyboard navigation doesn't work as expected either.*

Does this issue occur when all extensions are disabled?: 
Yes

**Expected result**:
- Access keys are visible at all times, i.e. even before pressing the Alt key.

After pressing the Alt key:
- First menu item is focused and access keys remain visible when Alt key is released.
- When pressing an access key, the corresponding menu option is triggered/opened.
- When opening a menu item with the arrow keys and then pressing an access key, triggers/opens menu item.

**Actual result**:
- Access keys are not visible before pressing the Alt key.

After pressing the Alt key:
- First menu item is focused, but access keys immediately become invisible when Alt key is released.
- Pressing access key doesn't do anything.
- When opening a menu item with the arrow keys, access keys are visible again, but pressing them does nothing.

Only when holding the Alt key the entire time, keyboard navigations works as expected.

**Animated GIF's to compare behaviour with other Windows apps**:
See the difference between Visual Studio Code and Visual Studio (the latter works as expected, although you can basically use any classic Windows application, for example Notepad works the same way too).

Visual Studio Code:
![vscodeaccesskeys](https://user-images.githubusercontent.com/30838294/42735846-cc46a038-885b-11e8-821e-eca5b0c30dd2.gif)
**Note**: I pressed down instead of the access key 'F' for *File* because the access key doesn't work.

Visual Studio:
![vsaccesskeys](https://user-images.githubusercontent.com/30838294/42735880-6b9e4564-885c-11e8-812c-2f363eb72e45.gif)

**Comment by sbatten** [comment]
@rik-smeets I believe nearly everything in this issue has been addressed with the exception of honoring the system setting to show underlines at all times. Thanks for pointing this out.

**Comment by rik-smeets** [comment]
@sbatten Just checked with the latest Insider build and I can verify that, awesome work!

I would however propose to keep this issue open until the system setting to show underlines at all times is honoured as well, I think that's the last hurdle in keyboard accessibility for the new menu bar. 

(But again: it's so much better already than the old one, thanks 👍)

**Comment by rik-smeets** [comment]
@sbatten Did find another issue: if there are duplicate access keys, only the first one is shown. This is currently the case in the `View` menu, where both `Debug console` and `Toggle Breadcrumb` have the `b` key set as the access key. Pressing it triggers `Debug console`. 

The expected behaviour is: when there are duplicate values, the menu options with matching access keys should be cycled through with each key press. User can than activate it with the Enter key.

**Comment by sbatten** [comment]
@rik-smeets sounds good. for the breadcrumbs issue, this is actually specific to `Toggle Breadcrumbs`. The behavior you described as expected is implemented. Try Alt+V+A+Z, View > Appearance > Zoom In/Out. I'll fix the breadcrumbs issue

**Comment by sbatten** [comment]
To verify, on Windows, enable custom titlebar turn on this setting on Windows:
![image](https://user-images.githubusercontent.com/6561887/53164328-40508800-35d0-11e9-8bf3-e4f189053ff7.png)

VS Code must be launched with the setting enabled. Verify that mnemonics are always visible in the menu.

**Comment by rik-smeets** [comment]
@sbatten Looks good to me, thanks!
